### PR TITLE
Replacing the Application Architecture Image

### PR DESCRIPTION
**Pull request by davidrichards-da**
CHANGELOG_BEGIN
CHANGELOG_END

Replacing the Application Architecture Image for an updated version and making a few small changes in to the copy.

### Pull Request Checklist

- [x] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/main/CONTRIBUTING.md)
- [ ] Include appropriate tests
- [x] Set a descriptive title and thorough description
- [ ] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [x] Include changelog additions in one or more commit message bodies between the `CHANGELOG_BEGIN` and `CHANGELOG_END` tags
- [ ] Normal production system change, include purpose of change in description
- [ ] If you mean to change the status of a component, please make sure you keep [the Component Status page](https://github.com/digital-asset/daml/blob/main/docs/source/support/component-statuses.rst) up to date.

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

File: docs/source/app-dev/recommended_architecture.svg [hunk 1]
```diff
@@ -1 +1 @@
...
...72839,94.87654320987656,0)" xlink:href="#W"/><use transform="matrix(0.06172839506172839,0,0,0.06172839506172839,101.0493827160494,0)" xlink:href="#an"/></g><path fill="#333" d="M30 0v-248h187v28H63v79h144v27H63v87h162V0H30" id="aK"/><path fill="#333" d="M141 0L90-78 38 0H4l68-98-65-92h35l48 74 47-74h35l-64 92 68 98h-35" id="aL"/><g id="P"><use transform="matrix(0.06172839506172839,0,0,0.06172839506172839,0,0)" xlink:href="#aK"/><use transform="matrix(0.06172839506172839,0,0,0.06172839506172839,14.814814814814813,0)" xlink:href="#aL"/><use transform="matrix(0.06172839506172839,0,0,0.06172839506172839,25.925925925925924,0)" xlink:href="#W"/><use transform="matrix(0.06172839506172839,0,0,0.06172839506172839,32.098765432098766,0)" xlink:href="#ac"/><use transform="matrix(0.06172839506172839,0,0,0.06172839506172839,44.44444444444444,0)" xlink:href="#aa"/><use transform="matrix(0.06172839506172839,0,0,0.06172839506172839,51.79012345679012,0)" xlink:href="#Y"/><use transform="matrix(0.06172839506172839,0,0,0.06172839506172839,64.1358024691358,0)" xlink:href="#V"/><use transform="matrix(0.06172839506172839,0,0,0.06172839506172839,76.48148148148148,0)" xlink:href="#S"/></g></defs></g></svg>
```
Comment (cocreature) [inline docs/source/app-dev/recommended_architecture.svg:1]: You dropped the path from the daml model to the DAR file. Was that intentional? Given how central your daml model is, it seems weird that it’s not not connected to anything.

File: docs/source/app-dev/app-arch.rst [hunk 2]
```diff
@@ -10,15 +10,13 @@ This section describes our recommended design of a full-stack Daml application.
...
-sections. Note that the Participant Node is integrated into the Daml drivers in some cases rather
-than being part of the Application Backend. See :doc:`/support/overview` for more details.
+The above image shows the recommended architecture. Of course there are many ways that the architecture and technology 
+stack can be changed to fit your needs which we'll mention in the corresponding sections.
 
 To get started quickly with the recommended application architecture, generate a new project using the ``create-daml-app`` template:
 
 .. code-block:: bash
-
+
```
Comment (cocreature) [inline docs/source/app-dev/app-arch.rst:19]:
```suggestion

```

File: docs/source/app-dev/recommended_architecture.svg (hunk 1 above)
Comment (davidrichards-da) [reply docs/source/app-dev/recommended_architecture.svg:1]: That was not intentional. Good catch. Ugh, sorry. Let me add it in.

File: docs/source/app-dev/app-arch.rst [hunk 3]
```diff
@@ -10,15 +10,12 @@ This section describes our recommended design of a full-stack Daml application.
...
-the architecture and technology stack to fit your needs, which we'll mention in the corresponding
-sections. Note that the Participant Node is integrated into the Daml drivers in some cases rather
-than being part of the Application Backend. See :doc:`/support/overview` for more details.
+The above image shows the recommended architecture. Of course there are many ways that the architecture and technology 
+stack can be changed to fit your needs which we'll mention in the corresponding sections.
 
 To get started quickly with the recommended application architecture, generate a new project using the ``create-daml-app`` template:
 
 .. code-block:: bash
-
```
Comment (cocreature) [inline docs/source/app-dev/app-arch.rst:21]: Sorry I wasn’t trying to delete the line but to delete the space you added. You need the empty line.

File: docs/source/app-dev/app-arch.rst [hunk 4]
```diff
@@ -10,15 +10,12 @@ This section describes our recommended design of a full-stack Daml application.
...
-The above image shows the recommended architecture. Of course there are many ways how you can change
-the architecture and technology stack to fit your needs, which we'll mention in the corresponding
-sections. Note that the Participant Node is integrated into the Daml drivers in some cases rather
-than being part of the Application Backend. See :doc:`/support/overview` for more details.
+The above image shows the recommended architecture. Of course there are many ways that the architecture and technology 
+stack can be changed to fit your needs which we'll mention in the corresponding sections.
 
 To get started quickly with the recommended application architecture, generate a new project using the ``create-daml-app`` template:
 
 .. code-block:: bash
```
Comment (cocreature) [inline docs/source/app-dev/app-arch.rst:18]:
```suggestion
.. code-block:: bash

```

**Pull request merged**